src/utils/storage.js

```diff
@@ -0,0 +1,14 @@
+import AsyncStorage from "@react-native-community/async-storage";
+
+export let player = null;
```

Comment:
Why do you need to export `player` ?

Reply:
I use it actually to send the current player ID inside API call's body.

Reply:
Mouaip, can do better  :laughing: As to discuss orally, this will be in a future PR